nats.go

```diff
@@ -2908,7 +2941,14 @@ func (nc *Conn) processMsg(data []byte) {
 	}
 
 	// FIXME(dlc): Should we recycle these containers?
-	m := &Msg{Header: h, Data: msgPayload, Subject: subj, Reply: reply, Sub: sub}
+	m := &Msg{
+		Subject: subj,
+		Reply:   reply,
+		Header:  h,
+		Data:    msgPayload,
+		Sub:     sub,
+		wsz:     len(data) + len(subj) + len(reply),
```

Comment:
If this is going to be used for maxbytes in pull consumer, then I think that you need to also add the header length (that the server uses to count towards max bytes). The header length should be nc.ps.ma.hdr as shown just above.

Reply:
I thought `data` here would have accounted both headers + data?

Reply:
Right, sorry about that.